code/datums/action.dm

```diff
@@ -873,8 +873,6 @@
 
 /datum/action/small_sprite/megafauna
 	icon_icon = 'icons/mob/actions/actions_xeno.dmi'
-	button_icon_state = "smallqueen"
-	background_icon_state = "bg_alien"
```

Comment:
Why did you remove these?

Reply:
Because it's a subtype of `/datum/action/small_sprite`, which has these vars already defined as these. No need to do it twice.

Reply:
oh yes i didn't see that